test/api_integration/apis/home/sample_data.ts

```diff
@@ -72,8 +73,14 @@ export default function ({ getService }: FtrProviderContext) {
           });
         });
 
-        // FLAKY: https://github.com/elastic/kibana/issues/166572
-        describe.skip('dates', () => {
+        describe('dates', () => {
+          // dates being compared are not arbitrary, but rather the dates of the earliest and latest timestamp of the flight sample data
+          // this can be verified in the flight data archive here {@link src/plugins/home/server/services/sample_data/data_sets/flights/flights.json.gz}
+          const sampleDataTimeIntervalInMS = differenceInMilliseconds(
+            new Date('2018-02-11T14:54:34'),
+            new Date('2018-01-01T00:00:00')
+          );
+
```

Comment:
The previous expectation was capped at exactly the value of 5 weeks (the actually difference in weeks between the data provided by the flight sample data) doesn't consider that whilst it really is 5 weeks there's some fringe time lost. Opting for this approach considers the entire time interval between the sample data provided and most be updated if the sample data is ever modified to a smaller or larger time range.

Reply:
curious why'd not subtract one d1.getTime() - d2.getTime()

Reply:
fair point... I totally didn't think about that, I mostly default to date fns helpers and given it was a test file, there was also no bundle size concern.